app/docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
       # For verifying client certificates
       - PHOENIX_TLS_VERIFY_CLIENT=False
       # In case we want to disable PII Redaction
-      - REDACT_PII=${REDACT_PII}
+      - REDACT_PII=${REDACT_PII:-True}
```

Comment:
Addresses `WARN[0000] The "REDACT_PII" variable is not set. Defaulting to a blank string.`